### PR TITLE
Update F# VSCode docs to be based off modern ionide

### LINKED ISSUE #9650: ClalssLibDemo fails
Just following the steps in this guide, creating a project in VSCode fails with these errors when either "Start Debugging" or "Start without Debugging" is run:  
Project parsing failed: ClassLibraryDemo.fsproj
Unable to open 'tasks.json': File not found.

Both fsi and fsc run from the command line, and Ionide has been uninstalled and reinstalled. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c543c271-0800-4ff9-4dfb-5154efb85732
* Version Independent ID: f1a4752f-a001-5e5e-2604-358db6f5adbc
* Content: [Get Started with F# in Visual Studio Code](https://docs.microsoft.com/en-us/dotnet/fsharp/get-started/get-started-vscode)
* Content Source: [docs/fsharp/get-started/get-started-vscode.md](https://github.com/dotnet/docs/blob/master/docs/fsharp/get-started/get-started-vscode.md)
* Product: **dotnet-fsharp**
* GitHub Login: @cartermp
* Microsoft Alias: **phcart**

### PR DESCRIPTION
**Pull request by cartermp**
Fixes #9650 and should keep issues like https://github.com/ionide/ionide-vscode-fsharp/issues/1214 from getting filed

### CODE REVIEW

**Comment by cartermp** [comment]
@baronfel @Krzysztof-Cieslak please take a look

File: docs/fsharp/get-started/get-started-vscode.md [hunk 1]
```diff
@@ -5,54 +5,40 @@ ms.date: 12/23/2018
...
 
-If you followed the previous step, you should get the Visual Studio Code Workspace on the left-hand side to appear with the following:
+## Configure F# interactive
 
-1. The F# project itself, underneath the `ClassLibraryDemo` folder.
-2. The correct directory structure for adding packages via [`Paket`](https://fsprojects.github.io/Paket/).
-3. A cross-platform build script with [`FAKE`](https://fsharp.github.io/FAKE/).
-4. The `paket.exe` executable that can fetch packages and resolve dependencies for you.
-5. A `.gitignore` file if you wish to add this project to Git-based source control.
+You'll first need to ensure that .NET Core scripting is your default scripting environment.
```
Comment (Krzysztof-Cieslak) [inline docs/fsharp/get-started/get-started-vscode.md]: Maybe we need to add some explanation why this is not default?

File: docs/fsharp/get-started/get-started-vscode.md [hunk 2]
```diff
@@ -129,63 +115,49 @@ Check if the first character of the input word is a vowel. If it is, attach "yay
...
-To begin, open `ClassLibraryDemo.fs`.  You'll notice that some code is already in there. Go ahead and delete the class definition, but make sure to leave the [`namespace`](../language-reference/namespaces.md) declaration at the top.
+To begin, open `Program.fs` file that you created earlier with the .NET CLI.  You'll notice that some code is already in there.
 
-Next, create a new [`module`](../language-reference/modules.md) called `PigLatin` and copy the `toPigLatin` function into it as such:
+Next, create a new [`module`](../language-reference/modules.md) called `PigLatin` and copy the `toPigLatin` function you created earlier into it as such:
 
 [!code-fsharp[ToPigLatin](~/samples/snippets/fsharp/getting-started/pig-latin.fs#L1-L14)]
 
-Next, open the `Script.fsx` file again, and delete the entire `toPigLatin` function, but make sure to keep the following two lines in the file:
+This module should be above the `main` function and below the `open System` declaration.
```
Comment (Krzysztof-Cieslak) [inline docs/fsharp/get-started/get-started-vscode.md]: Maybe some additional note about order of declarations being important in F#?

File: docs/fsharp/get-started/get-started-vscode.md [hunk 3]
```diff
@@ -129,63 +115,49 @@ Check if the first character of the input word is a vowel. If it is, attach "yay
...
-6. If Ionide is broken on your machine and none of the above has fixed your problem, try removing the `ionide-fsharp` directory on your machine and reinstall the plugin suite.
-
-Ionide is an open source project built and maintained by members of the F# community. Please report issues and feel free to contribute at the [Ionide-VSCode: FSharp GitHub repository](https://github.com/ionide/ionide-vscode-fsharp).
+1. If you've made changes to your system or installed Ionide prerequisites with Visual Studio Code open, restart Visual Studio Code.
+1. If you have invalid characters in your project directories, Ionide might not work.  Rename your project directories if this is the case.
+1. If none of the Ionide commands are working, check your [Visual Studio Code keybindings](https://code.visualstudio.com/docs/customization/keybindings#_customizing-shortcuts) to see if you're overriding them by accident.
+1. If Ionide is broken on your machine and none of the above has fixed your problem, try removing the `ionide-fsharp` directory on your machine and reinstall the plugin suite.
```
Comment (Krzysztof-Cieslak) [inline docs/fsharp/get-started/get-started-vscode.md]: Add a point about checking if project is successfully loaded, and ability to right click -> see details when project loading failed to get info about the problem

File: docs/fsharp/get-started/get-started-vscode.md [hunk 4]
```diff
@@ -129,63 +115,49 @@ Check if the first character of the input word is a vowel. If it is, attach "yay
...
 
-If you have an issue to report, please follow [the instructions for getting logs to use when reporting an issue](https://github.com/ionide/ionide-vscode-fsharp#how-to-get-logs-for-debugging--issue-reporting).
+Ionide is an open source project built and maintained by members of the F# community. Please report issues and feel free to contribute at the [ionide-vscode-fsharp GitHub repository](https://github.com/ionide/ionide-vscode-fsharp).
```
Comment (Krzysztof-Cieslak) [inline docs/fsharp/get-started/get-started-vscode.md]: Mentioning Open Collective would be too much, wouldn’t it?

**Comment by Krzysztof-Cieslak** [comment]
I would also love to see a section on debugging ;-)

File: docs/fsharp/get-started/get-started-vscode.md (hunk 4 above)
Comment (cartermp) [reply docs/fsharp/get-started/get-started-vscode.md]: Probably, yes.

**Review by Krzysztof-Cieslak — APPROVED** [review]

File: docs/fsharp/get-started/get-started-vscode.md [hunk 5]
```diff
@@ -5,60 +5,48 @@ ms.date: 12/23/2018
...
 ```console
-> F# new project
+dotnet new console -lang F# -o FirstIonideProject
 ```
 
-This is powered by the [FORGE](https://github.com/fsharp-editing/Forge) project.
-
-> [!NOTE]
-> If you don't see template options, try refreshing templates by running the following command in the Command Palette: `>F#: Refresh Project Templates`.
+Once it it completes, change directory to the project and open Visual Studio Code:
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]:
```suggestion
Once it completes, change directory to the project and open Visual Studio Code:
```

File: docs/fsharp/get-started/get-started-vscode.md [hunk 6]
```diff
@@ -5,60 +5,48 @@ ms.date: 12/23/2018
...
-
-## Creating your first project with Ionide
-
-To create a new F# project, open Visual Studio Code in a new folder (you can name it whatever you like).
+## Create your first project with Ionide
 
-Next, open the command palette (**View > Command Palette**) and type the following:
+To create a new F# project, open a command-line and create a new project with the .NET CLI:
 
 ```console
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]:
```suggestion
```dotnetcli
```

File: docs/fsharp/get-started/get-started-vscode.md [hunk 7]
```diff
@@ -5,60 +5,48 @@ ms.date: 12/23/2018
...
-> [!NOTE]
-> Ionide will generate .NET Framework F# projects, not dotnet core, which can have cross-platform compatibility issues. If you are running on **Linux** or **OSX**, a simpler way to get started is to use the [command-line tools](get-started-command-line.md).
-
-## Creating your first project with Ionide
-
-To create a new F# project, open Visual Studio Code in a new folder (you can name it whatever you like).
+## Create your first project with Ionide
 
-Next, open the command palette (**View > Command Palette**) and type the following:
+To create a new F# project, open a command-line and create a new project with the .NET CLI:
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: usually we say a console window or terminal not a command line

```suggestion
To create a new F# project, open a command line and create a new project with the .NET Core CLI:
```

File: docs/fsharp/get-started/get-started-vscode.md [hunk 8]
```diff
@@ -5,60 +5,48 @@ ms.date: 12/23/2018
...
-Select "F#: New Project" by hitting **Enter**. This takes you to the next step, which is for selecting a project template.
-
-Pick the `classlib` template and hit **Enter**.
+```console
+cd FirstIonideProject
+code .
+```
 
-Next, pick a directory to create the project in. If you leave it blank, it uses the current directory.
+After a short while, you should see the F# Solution Explorer pane on the left-hand side of your window open. This means Ionide has successfully loaded the project you just created. You can write code in the editor before this point in time, but once this happens everything will be loaded.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: Seems a bit strange to give a time metric. Perhaps:

```suggestion
After the project loads on Visual Studio Code, you should see the F# Solution Explorer pane on the left-hand side of your window open. This means Ionide has successfully loaded the project you just created. You can write code in the editor before this point in time, but once this happens, everything has finished loading.
```

File: docs/fsharp/get-started/get-started-vscode.md [hunk 9]
```diff
@@ -5,60 +5,48 @@ ms.date: 12/23/2018
...
 
-Finally, name your project in the final step. F# uses [Pascal case](http://c2.com/cgi/wiki?PascalCase) for project names. This article uses `ClassLibraryDemo` as the name. Once you've entered the name you want for your project, hit **Enter**.
+## Configure F# interactive
 
-If you followed the previous step, you should get the Visual Studio Code Workspace on the left-hand side to appear with the following:
+You'll first need to ensure that .NET Core scripting is your default scripting environment.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]:
```suggestion
First, ensure that .NET Core scripting is your default scripting environment:
```

File: docs/fsharp/get-started/get-started-vscode.md [hunk 10]
```diff
@@ -5,60 +5,48 @@ ms.date: 12/23/2018
...
 
-1. The F# project itself, underneath the `ClassLibraryDemo` folder.
-2. The correct directory structure for adding packages via [`Paket`](https://fsprojects.github.io/Paket/).
-3. A cross-platform build script with [`FAKE`](https://fsharp.github.io/FAKE/).
-4. The `paket.exe` executable that can fetch packages and resolve dependencies for you.
-5. A `.gitignore` file if you wish to add this project to Git-based source control.
+1. Open the Visual Studio Code settings (**Code > Preferences > Settings**).
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]:
```suggestion
1. Open the Visual Studio Code settings (**Code** > **Preferences** > **Settings**).
```

File: docs/fsharp/get-started/get-started-vscode.md [hunk 11]
```diff
@@ -5,60 +5,48 @@ ms.date: 12/23/2018
...
+1. Search for the term **F# Script**.
+1. Click the checkbox that says **FSharp: use SDK scripts**.
 
-## Writing some code
+This is currently necessary due to some legacy behaviors in .NET Framework-based scripting that do not work with .NET Core scripting, and Ionide is currently striving for that backwards compatibility. In the future, .NET Core scripting will become the default.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]:
```suggestion
This is currently necessary due to some legacy behaviors in .NET Framework-based scripting that don't work with .NET Core scripting, and Ionide is currently striving for that backwards compatibility. In the future, .NET Core scripting will become the default.
```

Comment (mairaw) [reply docs/fsharp/get-started/get-started-vscode.md]: Could potentially be in a note?

File: docs/fsharp/get-started/get-started-vscode.md [hunk 12]
```diff
@@ -5,60 +5,48 @@ ms.date: 12/23/2018
...
 
-Open the *ClassLibraryDemo* folder.  You should see the following files:
+### Write your first script
 
-1. `ClassLibraryDemo.fs`, an F# implementation file with a class defined.
-2. `ClassLibraryDemo.fsproj`, an F# project file used to build this project.
-3. `Script.fsx`, an F# script file that loads the source file.
-4. `paket.references`, a Paket file that specifies the project dependencies.
+Once you've configured Visual Studio Code to use .NET Core scripting, navigate to the Explorer view in Visual Studio code and create a new file. Name it `MyFirstScript.fsx`.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]:
```suggestion
Once you've configured Visual Studio Code to use .NET Core scripting, navigate to the Explorer view in Visual Studio Code and create a new file. Name it *MyFirstScript.fsx*.
```

File: docs/fsharp/get-started/get-started-vscode.md [hunk 13]
```diff
@@ -5,60 +5,48 @@ ms.date: 12/23/2018
...
 
-Open `Script.fsx`, and add the following code at the end of it:
+Now add the following code to it:
 
 [!code-fsharp[ToPigLatin](~/samples/snippets/fsharp/getting-started/to-pig-latin.fsx)]
 
 This function converts a word to a form of [Pig Latin](https://en.wikipedia.org/wiki/Pig_Latin). The next step is to evaluate it using F# Interactive (FSI).
 
-Highlight the entire function (it should be 11 lines long). Once it is highlighted, hold the **Alt** key and hit **Enter**. You'll notice a window pop up below, and it should show something like this:
+Highlight the entire function (it should be 11 lines long). Once it is highlighted, hold the **Alt** key and hit **Enter**. You'll notice a terminal window pop up below, and it should show something like this:
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]:
```suggestion
Highlight the entire function (it should be 11 lines long). Once it's highlighted, hold the **Alt** key and hit **Enter**. You'll notice a terminal window pop up on the bottom of the screen, and it should look similar to this:
```

File: docs/fsharp/get-started/get-started-vscode.md [hunk 14]
```diff
@@ -129,63 +117,52 @@ Check if the first character of the input word is a vowel. If it is, attach "yay
...
 
-## Moving your script code into the implementation file
+## Turn the console app into a Pig Latin generator
 
 The previous sections in this article demonstrated a common first step in writing F# code: writing an initial function and executing it interactively with FSI. This is known as REPL-driven development, where [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) stands for "Read-Evaluate-Print Loop". It's a great way to experiment with functionality until you have something working.
 
 The next step in REPL-driven development is to move working code into an F# implementation file. It can then be compiled by the F# compiler into an assembly that can be executed.
 
-To begin, open `ClassLibraryDemo.fs`.  You'll notice that some code is already in there. Go ahead and delete the class definition, but make sure to leave the [`namespace`](../language-reference/namespaces.md) declaration at the top.
+To begin, open `Program.fs` file that you created earlier with the .NET CLI.  You'll notice that some code is already in there.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]:
```suggestion
To begin, open the *Program.fs* file that you created earlier with the .NET Core CLI. You'll notice that some code is already in there.
```

**Review by mairaw — APPROVED** [review]
LGTM. Left a few suggestions for you to consider.

**Pull request merged**